language/stdlib/modules/validator_operator_config.move

```diff
@@ -0,0 +1,41 @@
+// Error codes:
+// 1100 -> OPERATOR_ACCOUNT_DOES_NOT_EXIST
+// 1101 -> INVALID_TRANSACTION_SENDER
+// 1103 -> VALIDATOR_OPERATOR_NOT_CERTIFIED
+address 0x0 {
+
+module ValidatorOperatorConfig {
+    use 0x0::Association;
+    use 0x0::Transaction;
+
+    resource struct T {
+        is_certified: bool,
+    }
+
+    public fun initialize(validator_operator_address: address) {
```

Comment:
let's make this hidden, since it will be part of account creation later -- i.e., create it if it doesn't exist, otherwise move along...

Reply:
Where do you think this can be hidden?
In the validator_config we can create resource in the rotate_operator. But who will create resource here - for validator_operator_config?